src/components/wallet/Activities.js

```diff
@@ -84,7 +85,8 @@ const StyledContainer = styled.div`
 
 const Activities = ({ transactions, accountId, getTransactionStatus }) => {
     const [transactionHash, setTransactionHash] = useState()
-    const activityLoader = actionsPending(['GET_TRANSACTIONS', 'REFRESH_ACCOUNT_OWNER'])
+    const { balance } = useSelector(({account}) => account)
+    const activityLoader = actionsPending(['GET_TRANSACTIONS', 'REFRESH_ACCOUNT_OWNER']) || !balance.stakedBalanceMainAccount
```

Comment:
not clear why activity loader is tied to `account.balance.stakedBalanceMainAccount`

Reply:
Same here, now the activity box on the dashboard will show three dots, without it it was showing an empty box, so it was suggesting that you don't have any recent activities.